custom-prom.md

```diff
@@ -98,34 +98,74 @@ Visiting `<your-kubecost-endpoint>/diagnostics.html` provides diagnostics info o
 
 Common issues include the following:
 
-**Wrong Prometheus FQDN**: Evidenced by the following pod error message `No valid prometheus config file at ...` and the init pods hanging. We recommend running `curl <your_prometheus_url>/api/v1/status/config` from a pod in the cluster to confirm that your [Prometheus config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration-file) is returned. Here is an example, but this needs to be updated based on your pod name and Prometheus address:
+### Misconfigured Prometheus FQDN
+
+Evidenced by the following pod error message `No valid prometheus config file at ...` and the init pods hanging. We recommend running `curl <your_prometheus_url>/api/v1/status/config` from a pod in the cluster to confirm that your [Prometheus config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration-file) is returned. Here is an example, but this needs to be updated based on your pod name and Prometheus address:
 
 ```sh
-kubectl exec kubecost-cost-analyzer-<UID> -c cost-analyzer-frontend -n kubecost \
--- curl http://<your_prometheus_url>/api/v1/status/config
+kubectl exec -i -t -n kubecost \
+  $(kubectl get pod --namespace kubecost|grep analyzer -m1 |awk '{print $1}') \
+  -c cost-analyzer-frontend -- \
+  curl http://<your_prometheus_url>/api/v1/status/config
 ```
 
-> **Note**: In the above example, <your_prometheus_url> may include a port number and/or a custom path name, resulting in a url like, e.g., http://kubecost-prometheus-server.kubecost:9080/prometheus/api/v1/status/config.
+> **Note**: In the above example, `$(kubectl get pod --namespace kubecost|grep analyzer -m1 |awk '{print $1}')` simply finds the name of a cost analyzer pod. You can replace this with the pod name, example: `kubecost-cost-analyzer-5bc6947b94-58hmx`
+
+> **Note**: In the above example, <your_prometheus_url> may include a port number and/or namespace, example: `http://prometheus-operator-kube-p-prometheus.monitoring:9090/api/v1/status/config`
 
 If the config file is not returned, this is an indication that an incorrect Prometheus address has been provided. If a config file is returned from one pod in the cluster but not the Kubecost pod, then the Kubecost pod likely has its access restricted by a network policy, service mesh, etc.
 
-**Context Deadline Exceeded**: Network policies, Mesh networks, or other security related tooling can block network traffic between Prometheus and Kubecost which will result in the Kubecost scrape target state as being down in the Prometheus targets UI. To assist in troubleshooting this type of error you can use the `wget` command from within the Prometheus Server container to try and reach the Kubecost target manually. Note the "namespace" and "deployment" name in this command may need updated to match your environment, this example uses the default Kubecost Prometheus deployment.\
```

Comment:
Explanation:
For this, a better test is to see if kubecost can reach the prometheus service, no?
Updated to curl prometheus from cost-analyzer pod